source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -72,11 +77,16 @@ export default class NetworkStatusStore extends Store {
   @observable isNodeResponding = false; // Is 'true' as long we are receiving node Api responses
   @observable isNodeSubscribed = false; // Is 'true' in case node is subscribed to the network
   @observable isNodeSyncing = false; // Is 'true' in case we are receiving blocks and not stalling
-  @observable isNodeTimeCorrect = true; // Is 'true' in case local and global time are in sync
   @observable isNodeInSync = false; // 'true' if syncing & local/network blocks diff within limit
   @observable isNodeStopping = false; // 'true' if node is in `NODE_STOPPING_STATES` states
   @observable isNodeStopped = false; // 'true' if node is in `NODE_STOPPED_STATES` states
 
+  // NTP
+  @observable isNodeTimeCorrect = true; // Is 'true' in case local and global time are in sync
+  @observable isSystemTimeIgnored = false; // Tracks if NTP time checks are ignored
+  @observable numberOfNTPRechecks = 0; // Is 'true' in case local and global time are in sync
+  NTPRecheckInterval: TimeoutID;
```

Comment:
@daniloprates can you please define the interval like this: https://github.com/input-output-hk/daedalus/blob/a973d8bbfc7d045d1bb5f07ba18db5cd8f53c18c/source/renderer/app/stores/NetworkStatusStore.js#L69